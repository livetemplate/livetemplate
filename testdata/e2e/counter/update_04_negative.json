{
  "2": "-3",
  "3": "negative",
  "4": "<p>Counter is negative</p>",
  "5": "2023-01-01 10:20:00"
}
