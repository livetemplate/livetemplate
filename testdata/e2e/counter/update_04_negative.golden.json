{
  "1": "-3",
  "2": "negative",
  "3": "\u003cp\u003eCounter is negative\u003c/p\u003e",
  "4": "2023-01-01 10:20:00"
}